### ISSUE TITLE
Autofill Mission Page Function Broken

### ISSUE THREAD

**Issue by y2krazy** [alarofrunetotem/OrderHallCommander#23]
Clicking mission button autofill mission page => does not autofill mission.

![Wow_2023-01-22_14-17-19](https://user-images.githubusercontent.com/12468950/213938267-1753b058-ca3c-44ac-953b-b46b44d8f2d5.png)

![Wow_2023-01-22_14-17-29](https://user-images.githubusercontent.com/12468950/213938268-7328c41c-6891-49a5-b127-7adba45f42c4.png)

**Comment by dblanch369** [comment]
Yes this happened to me too.  Started with the recent patch (10.0.5)

**Comment by DougInOr** [comment]
sometimes still happening with 10.1.0
